### PR TITLE
Bluetooth: shell: gatt-write-without-response with repeat

### PR DESCRIPTION
**Pull request by cvinayak**
Add a repeat param to gatt-write-without-response so that
it covers what gatt-write-without-response-repeated was
doing. gatt-write-without-response was removed in the
commit 26eae70da.

gatt-write-signed too will have repeat param with this
change.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

### CODE REVIEW

File: subsys/bluetooth/shell/gatt.c [hunk 1]
```diff
@@ -379,8 +380,24 @@ int cmd_gatt_write_without_rsp(int argc, char *argv[])
...
-	err = bt_gatt_write_without_response(default_conn, handle,
-					     gatt_write_buf, len, sign);
+	repeat = 0;
+
+	if (argc > 4) {
+		repeat = strtoul(argv[4], NULL, 16);
+	}
+
+	if (!repeat) {
+		repeat = 1;
```
Comment (jhedberg) [inline subsys/bluetooth/shell/gatt.c]: Wouldn't it be simpler just to initialize repeat to 1 instead of 0 to start with?

**Review by jhedberg — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/shell/gatt.c (hunk 1 above)
Comment (jhedberg) [inline subsys/bluetooth/shell/gatt.c]: Well, this wouldn't cover the case when somebody passes '0' on the command line, but that's just user stupidity :)

Comment (cvinayak) [inline subsys/bluetooth/shell/gatt.c]: the strtoul could return 0, hence this if-clause covers that.

Comment (cvinayak) [inline subsys/bluetooth/shell/gatt.c]: without this check, putting 0 in console will lead to the printk displaying junk for err.

**Review by Vudentz — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**